cmake/kokkos_arch.cmake

```diff
@@ -57,6 +57,7 @@ DECLARE_AND_CHECK_HOST_ARCH(ICX               "Intel Ice Lake Server CPUs (AVX51
 DECLARE_AND_CHECK_HOST_ARCH(SKX               "Intel Sky Lake Xeon E-class HPC CPUs (AVX512)")
 DECLARE_AND_CHECK_HOST_ARCH(KNC               "Intel Knights Corner Xeon Phi")
 DECLARE_AND_CHECK_HOST_ARCH(KNL               "Intel Knights Landing Xeon Phi")
+DECLARE_AND_CHECK_HOST_ARCH(SPR               "Intel Sapphire Rapids Xeon Server CPUs (AVX512)")
```

Comment:
Please confirm we should be saying "server"

Reply:
Yeah its fine, its kind of doubled since Xeon in and of itself implies server but its also fine to say it.